opnode/contracts/Makefile

```diff
@@ -59,3 +82,19 @@ bindings-optimism-portal:
 		--pkg deposit \
 		--type OptimismPortal \
 		--out ./deposit/deposit_feed_raw.go
+
+
```

Comment:
Why exactly is this commented out?

Reply:
Because the L2 Output Oracle Contract is built in the `l2os` package. I can remove these fully.